src/applications/check-in/pre-check-in/tests/e2e/pages/NextOfKin.js

```diff
@@ -47,6 +47,9 @@ class NextOfKin {
   attemptToGoToNextPage() {
     cy.get('button[data-testid="yes-button"]').click();
   }
+  attemptToGoToNextPageNoButtonClicked() {
```

Comment:
Maybe make this an abstract these two functions into one. Something like 

```
attemptToGoToNextPage(button = "yes") {
    cy.get(`button[data-testid="${button}-button"]`).click();
}
```

Reply:
> Some minor feedback. Great work

Thanks @mdewey ! All great points. I work on getting these updated shortly.

Reply:
Updated in commit [1233dad](https://github.com/department-of-veterans-affairs/vets-website/pull/19570/commits/1233dad1e575ab03a669892a49d307c2801b2dbc)